### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by maelle**
cf #58, here's what I suggest

* Make README shorter, put search details into a vignette.
* Re-work pkgdown navbar and reference.
* Add extra.css otherwise the table of contents of the search vignette looks bad

In the local installed version users would see the package level docs, the vignette about search, and for other features I think the function index is enough (so no need to reproduce the README content as a vignette?)

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -27,162 +29,58 @@ options(max.print = 200)
...
-The search server uses the stems of the words in the indexed metadata, and
-the search phrase. This means that "colour" and "colours" deliver the
-exact same result. So do "coloring", "colored", etc. (Unless one is happen
-to be an exact package name or match another non-stemmed field.)
-
-```{r}
-ps("colour", size = 3)
-ps("colours", size = 3)
-```
+`pkgsearch` uses three R-hub web services ([crandb](https://github.com/r-hub/crandb), [cranlogs](https://github.com/r-hub/cranlogs.app) and [search](https://github.com/metacran/search)) that munge CRAN metadata and let you access it through several lenses.
```
Comment (gaborcsardi) [inline README.Rmd]: I want to be able to change the web service backend, so I would go into the details here.

File: README.Rmd [hunk 2]
```diff
@@ -27,162 +29,58 @@ options(max.print = 200)
...
 
-```{r}
-cran_packages()
-cran_packages(from = "pkgsearch")
+```r
+install.packages("pkgsearch")
 ```
 
-### List R releases in the CRANDB database
+Or the development version from GitHub:
```
Comment (gaborcsardi) [inline README.Rmd]: I think we should not advertise the installation of development versions at all.

Comment (maelle) [reply README.Rmd]: But won't this README be quite confusing pre-release with all the new stuff that's not on CRAN?

Comment (gaborcsardi) [reply README.Rmd]: That's just a temporary thing. But then maybe we can say above to "Install the latest pkgsearch release from CRAN:" etc.

**Comment by maelle** [comment]
The mp4 to gif conversion went wrong :thinking:

**Comment by maelle** [comment]
Not sure what to think of the LaTeX travis failure.

**Comment by maelle** [comment]
The GIF is much too big atm

**Pull request merged**